README.md

```diff
@@ -52,57 +54,48 @@ Smart Mastering Core is a community-supported project. Help is available by
 filing issues here on GitHub and by asking questions on Stack Overflow; 
 however, we can’t promise a specific resolution or timeframe for any request. 
 
-### Instructions
+### Adding Smart Mastering to your project
 
-In your project's `build.gradle` file, add this to the `buildscript` section:
+Assuming you're using ml-gradle, you can easily integrate Smart Mastering into your application.
 
-```
-// Needed for smart-mastering-core dependency until it's available via jcenter()
-maven {
-  url {"https://dl.bintray.com/marklogic-community/Maven/"}
-}
-```
+As this project hasn't been published to the [jcenter](https://bintray.com/bintray/jcenter) repository yet, you'll first
+need to publish a copy of this project to your local Maven repository, which defaults to ~/.m2/repository. 
 
-In the `configurations` section, add a line with `smartMasteringCore` all by
-itself.
+To do so, just run the following command in this directory:
```

Comment:
Should this say, "To do so, clone this repository and run the following command in the project's directory:"? (just to be clear about required steps)

Reply:
Yup, will change